fs/fs.go

```diff
@@ -78,9 +78,10 @@ func NewFsInfo(context Context) (FsInfo, error) {
 	}
 	supportedFsType := map[string]bool{
 		// all ext systems are checked through prefix.
-		"btrfs": true,
-		"xfs":   true,
-		"zfs":   true,
+		"btrfs":   true,
+		"overlay": true,
```

Comment:
so for now perhaps say if mount point is / and its overlay we dont ignore
it?

On Fri, Feb 26, 2016 at 2:43 PM, Vish Kannan <notifications@github.com>
wrote:

> In fs/fs.go
> <https://github.com/google/cadvisor/pull/1125#discussion_r54312699>:
>
> > @@ -78,9 +78,10 @@ func NewFsInfo(context Context) (FsInfo, error) {
> >  	}
> >  	supportedFsType := map[string]bool{
> >  		// all ext systems are checked through prefix.
> > -		"btrfs": true,
> > -		"xfs":   true,
> > -		"zfs":   true,
> > +		"btrfs":   true,
> > +		"overlay": true,
>
> Ideally docker root fs should be detected using docker's APIs (info) and a
> statfs on that should be good enough.
> This code just needs some love.
> This logic here was originally meant for discovering filesystems of
> interest at a global level.
> Layered filesystems that are ephemeral are not of interest to most users
> in the global context.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/1125/files#r54312699>.
>